src/Layer/GeometryLayer.js

```diff
@@ -104,7 +109,7 @@ class GeometryLayer extends Layer {
         this.wireframe = false;
 
         this.attachedLayers = [];
-        this.visible = config.visible == undefined ? true : config.visible;
+        this.visible = visible == undefined ? true : visible;
```

Comment:
It's more intuitive to use `Layer.visible` and more similar to other APIs.
`this.object3d.visible` is like a private property and `setter` is used for emit event.

Reply:
> ```js
> this.visible = this.visible ?? true;
> ```

small typo 
```js
this.visible = visible ?? true;
```

Reply:
> It's more intuitive to use `Layer.visible` and more similar to other APIs. `this.object3d.visible` is like a private property and `setter` is used for emit event.

Yep, I get that, but my point was that line 112 could be removed, since the getter and setter are set for `visible` property, and there is no usage of `this.visible`.




> > ```js
> > this.visible = this.visible ?? true;
> > ```
> 
> small typo
> 
> ```js
> this.visible = visible ?? true;
> ```

Indeed sorry, well spotted